sktime/forecasting/all/__init__.py

```diff
@@ -72,3 +110,51 @@
 from sktime.forecasting.online_learning._online_ensemble import (
     OnlineEnsembleForecaster,
 )
+from sktime.performance_metrics.forecasting import (
+    MeanAbsoluteScaledError,
+    MedianAbsoluteScaledError,
+    RootMeanSquaredScaledError,
+    RootMedianSquaredScaledError,
+    MeanAbsoluteError,
+    MeanSquaredError,
+    RootMeanSquaredError,
+    MedianAbsoluteError,
+    MedianSquaredError,
+    RootMedianSquaredError,
+    SymmetricMeanAbsolutePercentageError,
```

Comment:
Ah I see the problem of having abbreviations (sMAPEmay refer to mean or median). Is there any other way to handle this? Or could we include the abbreviations still even if we just redefine them `sMAPE =  SymmetricMeanAbsoluteError`, but duplication also isn't great ...

Reply:
Hey @mloning can you elaborate on your comment? 

I thought about using abbreviation sMAPE and sMdAPE, but I think we should match the metric naming style of scikit-learn regression metrics. Scikit-learn has functions that calculate MSE, MAPE and MdAE and are named mean_square_error, mean_absolute_percentage_error and median_absolute_error. 

Also in terms of duplication do you mean having a separate function for the median and mean metrics or something else?  

I actually coded a  `symmetric=False` default argument into the mean_absolute_percentage_error function and then have symmetric_mean_absolute_percentage_error as a wrapper around that implementation. I could go either way in terms whether we exclude the symmetric_mean_absolute_percentage_error function and just let users toggle the `symmetric` argument in mean_absolute_percentage_error based on their preference.

In terms of the separate functions for mean and median, I considered whether to include a `aggregation_function` parameter that let people choose {'max', 'mean', 'median'}. That would reduce the number of functions, but it would lead to a more ambiguous name (we'd have to name the function something like root_squared_error and then let people chose whether it is mean, median or max version).  It would also move away from using names that match the scikit-learn regression metric naming style.

Reply:
Okay @RNKuhns I'm happy with using the long names, makes it more explicit.

Reply:
We could simply add shorthands for common loss functions, e.g. `smape = symmetric_mean_absolute_error` so that users can also import `smape` - but not sure if the benefit of having shorter function names for common errors outweighs the disadvantage of introducing two ways for doing the same thing

Reply:
I agree, using arguments like {'max', 'mean', 'median'} isn't very intuitive.